### DISCUSSION TITLE
Lista pagamenti. E' obbligatoria?

### DISCUSSION THREAD

**Discussion by TRISolutions** [fattureincloud/api#382] [Q&A] [answered]
Perdonate la domanda banale, ho fatto in passato software per generare l'XML delle fatture elettroniche e in molti casi non ho mai inserito i singoli pagamenti.
Invece, utilizzando la SDK PHP non riesco a generare fatture senza valorizzarne la lista dei pagamenti. Ottengo sempre l'errore: "Il totale dei pagamenti non corrisponde al totale da pagare." 
Cosa sbaglio?
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @TRISolutions la nostra api obbliga a inserire almeno un pagamento che corrisponda al totale del documento (settabile anche come `not_paid`) nel caso in cui vengano specificati degli articoli (`items_list`), per questo le viene ritornato l'errore sui totali.

Grazie,
Emanuele.

**Comment by TRISolutions** [comment]
Grazie Emanuele per la risposta. 
Inserirò sempre la lista dei pagamenti come richiesto dalle API.
